padd.sh

```diff
@@ -1078,22 +1075,9 @@ PrintLogo() {
         printf "%s${clear_line}\n" "n${padd_text} ${mini_status}"
     elif [ "$1" = "micro" ]; then
         printf "%s${clear_line}\n${clear_line}\n" "µ${padd_text}     ${mini_status}"
-    elif [ "$1" = "mini" ]; then
```

Comment:
Keep the `elif [ "$1" = "mini" ]; then` here and change L1657-1659 in `StartupRoutine()` to call PrintLogo("${1}") which then can default to the `else` case printing the retro logo. This reduces the code a tiny bit and makes it more standardized.